lib/Tags/TagsForm.js

```diff
@@ -54,14 +56,16 @@ class TagsForm extends React.Component {
     if (!filter) {
       return { renderedItems: list };
     }
-
-    const filtered = list.filter(item => item.value.toLowerCase().includes(filter.toLowerCase()));
-    const renderedItems = filtered.sort((a, b) => {
-      if (a.value.toLowerCase().startsWith(filter.toLowerCase())) return -1;
-      if (b.value.toLowerCase().startsWith(filter.toLowerCase())) return 1;
-
-      return a.value.localeCompare(b.value);
-    });
+    const filterLowercased = filter.toLowerCase();
+
+    const renderedItems = [
+      ...list
+        .filter(({ value }) => value.toLowerCase().startsWith(filterLowercased))
+        .sort(localeCompareCB),
+      ...list
+        .filter(({ value }) => !value.toLowerCase().startsWith(filterLowercased) && value.toLowerCase().includes(filterLowercased))
+        .sort(localeCompareCB),
+    ];
```

Comment:
Shouldn't old variant be more effective?

Reply:
firstly, Old variant doesn't work right. so it might be more effective from memory or time consumption perspective, but it doesn't help.
secondly, I don't think that two filters instead of one makes so big difference